Sight_++_Solution/Intel_Sight_PP/src/caffe_impl.cpp

```diff
@@ -42,6 +43,10 @@ struct CaffeModelImpl : public ModelInterface {
 		auto input_blob = cv::dnn::blobFromImage(color_matrix, inScaleFactor, cv::Size(inWidth, inHeight), meanVal, false);
 		net.setInput(input_blob, "data");
 
+		net.setPreferableBackend(cv::dnn::DNN_BACKEND_OPENCV);
```

Comment:
Does this need to be done in each call to the do_work?
Does this not belong in the constructor?

Reply:
and the setPreferableTarget